rpm/__init__.py

```diff
@@ -64,16 +65,26 @@ def get_sitepackages_and_suffixes(interpreter):
     # try platform-python first (it could be the only interpreter present on the system)
     interpreters = [
         "/usr/libexec/platform-python",
-        f"/usr/bin/python{majorver}",
         f"/usr/bin/python{majorver}.{minorver}",
+        f"/usr/bin/python{majorver}",
     ]
     result = []
     for interpreter in interpreters:
         if not Path(interpreter).is_file():
             continue
+        # skip interpreters with a different major.minor version than the
+        # running interpreter
+        # importing their rpm module has most likely catastrophic consequences
```

Comment:
As was already established, since Python 3.7 a different minor version shouldn't have any consequences, let alone catastrophic.

Reply:
Yes, but the comment is still misleading.

Reply:
Better now?